docs/standard/parallel-programming/exception-handling-task-parallel-library.md

```diff
@@ -83,7 +83,14 @@ If a task completes in the <xref:System.Threading.Tasks.TaskStatus.Faulted?displ
 [!code-csharp[TPL_Exceptions#27](../../../samples/snippets/csharp/VS_Snippets_Misc/tpl_exceptions/cs/exceptionprop21.cs#27)]
 [!code-vb[TPL_Exceptions#27](../../../samples/snippets/visualbasic/VS_Snippets_Misc/tpl_exceptions/vb/exceptionprop21.vb#27)]
 
-In a real application, the continuation delegate could log detailed information about the exception and possibly spawn new tasks to recover from the exception.
+In a real application, the continuation delegate could log detailed information about the exception and possibly spawn new tasks to recover from the exception. If a task faults, the following expressions with throw the exception:
+
+ - `await task`
+ - `task.Wait()`
+ - `task.Result`
+ - `task.GetAwaiter().GetResult()`
+
+To observe the exception, access the <xref:System.Threading.Tasks.Task.Exception%2A> property.
```

Comment:
```suggestion
To observe the exception, access the <xref:System.Threading.Tasks.Task.Exception%2A?displayProperty=nameWithType> property.
```

Reply:
I didn't realize that was an option for `xref` syntax - I've never used it before joining the .NET docs team, thanks for sharing that!